pkgs/applications/networking/browsers/chromium/default.nix

```diff
@@ -183,7 +183,8 @@ in stdenv.mkDerivation {
     mkdir -p "$out/bin"
 
     eval makeWrapper "${browserBinary}" "$out/bin/chromium" \
-      --add-flags ${escapeShellArg (escapeShellArg commandLineArgs)}
+      --add-flags ${escapeShellArg (escapeShellArg commandLineArgs)} \
+      --add-flags "\''${NIXOS_OZONE_WL:+\''${WAYLAND_DISPLAY:+--enable-features=UseOzonePlatform --ozone-platform=wayland}}"
```

Comment:
`commandLineArgs` exists, might be better to use that?

Reply:
We shouldn't use `commandLineArgs` itself as that isn't extendable (we could only set a default) but we could use the same `escapeShellArg` based approach.